src/applications/edu-benefits/1990s/pages/application.js

```diff
@@ -0,0 +1,266 @@
+import fullSchema from 'vets-json-schema/dist/VRRAP-schema.json';
+import ssnUI from 'platform/forms-system/src/js/definitions/ssn';
+import fullNameUI from 'platform/forms/definitions/fullName';
+import currentOrPastDateUI from 'platform/forms-system/src/js/definitions/currentOrPastDate';
+import phoneUI from 'platform/forms-system/src/js/definitions/phone';
+import emailUI from 'platform/forms-system/src/js/definitions/email';
+import bankAccountUI from 'platform/forms/definitions/bankAccount';
+
+import {
+  uiSchema as addressUISchema,
+  schema as addressSchema,
+} from 'platform/forms/definitions/address';
+import _ from 'lodash';
+import {
+  bankInfoHelpText,
+  directDepositAlert,
+  directDepositDescription,
+} from '../content/directDeposit';
+
+import {
+  confirmEligibilityDescription,
+  confirmEligibilityNote,
+} from '../content/confirmEligibility';
+
+const {
+  veteranFullName,
+  veteranSocialSecurityNumber,
+  dateOfBirth,
+  email,
+  mobilePhone,
+  alternatePhone,
+  hasSelectedProgram,
+  providerName,
+  programName,
+  programCity,
+  programState,
+  learningFormat,
+  vrrapConfirmation,
+} = fullSchema.properties;
+
+const { bankAccount } = fullSchema.definitions;
+
+const addressUiSchema = addressUISchema('Mailing address', false);
+const address = addressSchema(fullSchema, true);
+const declineDirectDeposit = form =>
+  form['view:directDeposit']['view:declineDirectDeposit'];
+const useDirectDeposit = form => !declineDirectDeposit(form);
+const hasNotSelectedProgram = form =>
+  !_.get(form['view:programSelection'], 'hasSelectedProgram', true);
+
+const path = 'form';
+const title = 'Application';
+const uiSchema = {
+  'view:applicantInformation': {
+    'ui:title': 'Applicant Information',
+    veteranFullName: fullNameUI,
+    veteranSocialSecurityNumber: ssnUI,
+    dateOfBirth: {
+      ...currentOrPastDateUI('Date of birth'),
+      'ui:errorMessages': {
+        required: 'Please provide a valid date',
+        futureDate: 'Please provide a valid date',
```

Comment:
Should this be something like 'Please provide a valid date in the past'?

Reply:
Copy and paste from 0994, will update